CityOfDetroit/bloom
# feat: occupancy table based on unit groups

## Issue

- Closes #990
- Addresses #issue

## Description

Occupancy tables based on unit groups - each row is its own unit group

Removes the dynamic string above the occupancy table that changed based on whether or not there are SRO units (requested by Kathy, those strings are not relevant here)

## Type of change


## How Can This Be Tested/Reviewed?

I updated the `Martin Luther King II` listing to include unit groups with the different possibilities of occupancy/unit types for this table. The rows are sorted on num bedrooms of the first unit type. In current core not all of these strings are translated. We should pull the translation fixes here back into core.

For the unit type string:
* You can have one type or multiple - for multiple they are comma separated and sorted on num bedrooms

For the occupancy string:
* If both min and max are null no row shows
* If min === max shows no range just one number
* If !min and max, the limit is at most the max number
* If !max and min, the limit is at least the min number
* Otherwise show a range

<img width="320" alt="Screen Shot 2022-03-07 at 7 56 16 PM" src="https://user-images.githubusercontent.com/65367387/157157425-a51f7403-d31a-41a2-a691-f6c0aaaed0ec.png">


## Checklist:
